src/File.php

```diff
@@ -46,6 +46,9 @@ class File
     /** @var int */
     protected $fileSize;
 
+    /** @var string[] */
+    private $metaInfo = [];
```

Comment:
@ankitpokhrel Thanks for the feedback. I've incorporated your suggestions in pushed commits (9176f004a1448d65d3258b9c42d1a17261a93239, 0538493c84a887c9c51662409394a300e0cf7954).

1. I've decided to keep `extractAllMeta` instead of `getAllMeta` to keep it similar to `extractMeta`. Also there is an operation performed during it, so it's not a simple getter.
2. I've remembered to remove additional instructions from `Server::buildFile` (since metadata is now under separate key and set outside of this method).

Reply:
@ankitpokhrel please take a look at the test changes, just in case I've missed any side effects that might break something. It's kinda late at night when I'm adjusting this :)